src/block_service/directio_writable_file.h

```diff
@@ -0,0 +1,54 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#pragma once
+
+#include <string>
+
+namespace dsn {
+namespace dist {
+namespace block_service {
+
+class direct_io_writable_file
+{
+public:
+    explicit direct_io_writable_file(const std::string &file_path);
+    ~direct_io_writable_file();
+
+    // can't copy or move
+    direct_io_writable_file(const direct_io_writable_file &) = delete;
+    direct_io_writable_file &operator=(const direct_io_writable_file &) = delete;
+    direct_io_writable_file(direct_io_writable_file &&) = delete;
+    direct_io_writable_file &operator=(direct_io_writable_file &&) = delete;
```

Comment:
How about use `DISALLOW_COPY_AND_ASSIGN` instead to keep the same style?

Reply:
Sure, tks